web/client/components/style/ThemaClassesEditor.jsx

```diff
@@ -129,6 +153,38 @@ class ThemaClassesEditor extends React.Component {
         }
     };
 
+    updateClassification = (classIndex, type) => {
+        let updateIndex;
+        let updateMinMax;
+        let deleteCount = 0;
+        let newClassification = this.props.classification;
```

Comment:
I think we need to copy this array if not the reference to this list of entry will break the undo/redo functionalities
```suggestion
        let newClassification = [...this.props.classification];
```